### PR TITLE
fix: standarize sidebar headers

### PR DESCRIPTION
**Pull request by tomaszantas**
## Description

Standardize headings in sidebars.

## How this has been tested?

image

image

image

image

image

image

### CODE REVIEW

File: src/components/ProfitAndLossDetailedCharts/ProfitAndLossDetailedCharts.tsx [hunk 1]
```diff
@@ -66,6 +66,14 @@ export const ProfitAndLossDetailedCharts = ({
 
       <header className='Layer__profit-and-loss-detailed-charts__header--tablet'>
         <BackButton onClick={() => setSidebarScope(undefined)} />
+        <div className='Layer__profit-and-loss-detailed-charts__head'>
+          <Text size={TextSize.lg} weight={TextWeight.bold} className='title'>
```
Comment (ArkuVonSymfon) [inline src/components/ProfitAndLossDetailedCharts/ProfitAndLossDetailedCharts.tsx:70]: Should this className be just "title" or "date" ? Not like "Layer__title"?

File: src/styles/details_list.scss [hunk 2]
```diff
@@ -3,6 +3,16 @@
   flex-direction: column;
 }
 
+.Layer__details-list__actions > .Layer__details-list__close-btn {
+  width: 36px;
+  height: 36px;
+  padding: 0;
+
+  @container (max-width: 1025px) {
```
Comment (ArkuVonSymfon) [inline src/styles/details_list.scss:11]: most small viewports have 1024 should we use 1025?

**Review by ArkuVonSymfon — APPROVED** [review]
Left some comments to double-check if that is correct

File: src/components/ProfitAndLossDetailedCharts/ProfitAndLossDetailedCharts.tsx (hunk 1 above)
Comment (tomaszantas) [reply src/components/ProfitAndLossDetailedCharts/ProfitAndLossDetailedCharts.tsx:70]: I would go with plain `title` and `date`. `Layer__title` indicates on more globally used component.

File: src/styles/details_list.scss (hunk 2 above)
Comment (tomaszantas) [reply src/styles/details_list.scss]: Fixed, thanks for catching this!

**Pull request merged**